teaser/logic/buildingobjects/useconditions.py

```diff
@@ -52,13 +52,11 @@ class UseConditions(object):
         Degree Days and Precipitation in Europe—analysis for the
         CELECT-project" by Benestad, 2008.
     heating_profile : list [K]
-        Heating setpoint for a day or similar. You can set a list of any
+        Heating setpoint of AHU for a day or similar. You can set a list of any
```

Comment:
this is not correct, this heating_profile is meant for the static ideal heating

Reply:
Thank you for clarifying! Now I wonder, why we can set the ideal heating threshold in 3 different ways: T_threshold_heating, heating_profile, and eventually schedule?

Does T_threshold_heating regard to the outside temperature? And marks a general shut-down of the component unregarding the zone temperature?

Do you know how the attribute schedule works? I think it could need a few more explaining words as well. If you comment some here, I can add them.

Reply:
> Does T_threshold_heating regard to the outside temperature? And marks a general shut-down of the component unregarding the zone temperature?

Correct!

> Do you know how the attribute schedule works? I think it could need a few more explaining words as well. If you comment some here, I can add them. 

Attribute schedule is private and should not be used directly (thus has no documentation) There are only two attributes for indoor temperature `heating_profile`, `cooling_profile`

Reply:
Thank you again for clarifying!

Please correct me, but in that case, there is no attribute to define the supply air temperature of the AHU within the JSON. So the supply temperature has to be set via central_ahu.temperature_profile within python. Which makes sense, because this is a building-related parameter and not a zone related one.

Do you know how the use_conditions.min_ahu/max_ahu relate with central_ahu.v_flow_profile?

As I could find out there is a parameter "withprofile" in modelica which defines whether the SUP flow is calculated via:

- withprofile = true -> via central_ahu.profile and min_ahu/max_ahu

- withprofiile = false -> rel. occupancy and min_ahu/max_ahu

But I was not able to find where this parameter "withprofile" is set.

Auszug Modelica: 
block AixLib.ThermalZones.ReducedOrder.Multizone.BaseClasses.AirFlowRateSum 
  "Air flow rate for AHU and unit conversion"
  extends Modelica.Blocks.Icons.Block;

  parameter Integer dimension "Number of Zones";
  parameter Boolean withProfile=false 
    "Profile or occupancy as control value for AHU";

After I understood the behavior, I will extend the documentation.

Reply:
I cannot find "WithProfile" in teaser\data\output\modelicatemplate\AixLib\AixLib_Multizone
I cannot find it somewhere else in AixLib where it could be overwritten.
I cannot find a parameter in Teaser that could define this "WithProfile"

At my current state of research I suspect that calculations are always done via the rel. occupancy and the AHU profile has no effect. (Except someone changes the "WithProfile" value within Modelica)

Reply:
I think this is due to the fact that withProfile is by default=true in the Model `AixLib.ThermalZones.ReducedOrder.Multizone.MultizoneEquipped` that is used in TEASER export. thus i believe the v_flow_profile and min/max_ahu correlate directly in a sense that the maximum air volume supplied to the zone is:

```Modelica
((zoneParam.minAHU + (zoneParam.maxAHU -
    zoneParam.minAHU) * profile) .* zoneParam.AZone)

Reply:
> Please correct me, but in that case, there is no attribute to define the supply air temperature of the AHU within the JSON. So the supply temperature has to be set via central_ahu.temperature_profile within python. Which makes sense, because this is a building-related parameter and not a zone related one.

that is correct

> After I understood the behavior, I will extend the documentation.

That discussion and the extension help a lot :) thanks

Reply:
Thanks for evaluating my thoughts!

I think the default withProfile value is false!? (current development of AixLib)
So it is always occupancy controlled.

I would open an issue and link this discussion to it.
For this PR I would leave it with the docstring extension.

Reply:
> I think the default withProfile value is false!? (current development of AixLib)
So it is always occupancy controlled.

I don't think so: https://github.com/RWTH-EBC/AixLib/blob/master/AixLib/ThermalZones/ReducedOrder/Multizone/MultizoneEquipped.mo#L133 (master - but dev should be the same)

Reply:
I was wrong. I checked the base class, multizoneEquipped overwrites this value. 

I will change everything accordingly.